templates/base.html.twig

```diff
@@ -9,6 +9,15 @@
             <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.9/css/all.css" integrity="sha384-5SOiIsAziJl6AWe0HWRKTXlfcSHKmYV4RBF18PPJ173Kzn7jzMyFuTtk8JA7QQG1" crossorigin="anonymous">
             <link rel="stylesheet" href="{{ asset('build/css/app.css') }}">
         {% endblock %}
+        <!-- Global site tag (gtag.js) - Google Analytics -->
```

Comment:
Est-ce que tu peux encapsuler ce code de {% if not app.debug %}...{% endif %} pour éviter d'avoir des remontées analytics en dev stp ?

Reply:
On essai aussi de placer les scripts à la fin, pour l'optim